app/actions/websocket/posts.ts

```diff
@@ -152,6 +152,7 @@ export async function handleNewPostEvent(serverUrl: string, msg: WebSocketMessag
                 myChannel.messageCount + 1,
                 myChannel.mentionsCount + (hasMentions ? 1 : 0),
                 myChannel.lastViewedAt,
+                myChannel.urgentMentionsCount,
```

Comment:
Of shouldn't hasMentions above consider urgent mentions as well?

Reply:
`urgent_mention_count` is already covered by `hasMentions`. If the current user is mentioned and the message is marked as urgent, their ID will still be included in `msg.data.mentions`. So `hasMentions` accounts for both regular and urgent mentions.